### ISSUE TITLE
lintian: source-contains-empty-directory

### ISSUE THREAD

**Issue by panlinux** [CanonicalLtd/ubuntu-advantage-client#281]
P: ubuntu-advantage-tools source: source-contains-empty-directory uaclient/__pycache__/

**Comment by powersj** [comment]
I think your build environment was not clean, as I cannot reproduce this and I do not see it on the Trusty package on the daily archive. There is no such folder in the repo either.

@panlinux can you confirm and close this please?

**Comment by panlinux** [comment]
Hm, it being a native package, the moment the source tarball is generated, it includes an empty __pycache__ directory:
```
$ tar tvJf ../ubuntu-advantage-tools_19.1-0ubuntu1.tar.xz|grep `__pycache__`
drwxrwxr-x 0/0               0 2019-04-05 20:47 ubuntu-advantage-client/uaclient/__pycache__/
```
I think we can ignore this for now, and just be careful when uploading for real.

**Comment by powersj** [comment]
I see this on the 19.2 PPA version as well

**Comment by powersj** [comment]
More info:

https://mail.python.org/pipermail/pypy-dev/2012-July/010369.html

**Comment by panlinux** [comment]
I know now what is going on, and it's a symptom of a deeper packaging problem. Working on it.

**Comment by powersj** [comment]
Fixed with a91a1974b38315b146d0ef329a06e131a701b68b
